modules/home-manager/shell-generic/files/helm-repo-browser.sh

```
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# helm-repo-browser - A script to browse Helm repositories and charts interactively.
+# Dependencies: helm, fzf, gum, bat
+
+set -euo pipefail
+
+# Colors and styling
+GUM_CONFIRM_STYLE="--selected.background 2 --unselected.background 0"
```

Shellcheck (0.11.0)

[warning] GUM_CONFIRM_STYLE appears unused. Verify use (or export if used externally).

(SC2034)